prov/psm/src/psmx_init.c

```diff
@@ -558,6 +558,7 @@ static int psmx_getinfo(uint32_t version, const char *node, const char *service,
 	psmx_info->domain_attr->mr_iov_limit = 65535;
 	psmx_info->domain_attr->caps = PSMX_DOM_CAPS;
 	psmx_info->domain_attr->mode = 0;
+	psmx_info->domain_attr->mr_cnt = 10000;
```

Comment:
> @jswaro (size_t)-1ULL is max value of size_t datatype.

That wasn't what I was asking. From the perspective of libfabric as an API, will we be considering -1 to mean 'unlimited'?

If I'm writing an application and I see that value, does that functionally mean I could register as many regions as I wanted (or more to the point, that the provider doesn't have a limit that it can easily express or nor does it have an informative limit)? My question is more directed toward how applications should interpret -1.